server/src/main/antlr4/org/eclipse/lsp/cobol/core/parser/CobolLexer.g4

```diff
@@ -48,6 +48,8 @@ ALSO : A L S O;
 ALTER : A L T E R;
 ALTERNATE : A L T E R N A T E;
 ALWAYS : A L W A Y S;
+ANA: A N A;
```

Comment:
There is a small issue we have not discussed yet. The newly added values are not defined by the COBOL specification, so they might be used as variable names. 
Please, create a new rule in the CobolParser below `cobolWord` named `dacoKeywords` and call from the `cobolWord`. You may take the `cobolKeywords` as an example. 
Then, please, put all the newly added keywords to this new rule separated by `|`.

Reply:
@temanbrcom When I add all newly added keywords to dacoKeywords then I am getting following error.
"Variable ITEM is not defined"

Reply:
Ok, I see. The problem is the intersection of the GET rules in the COBOL and in the DaCo grammars. This should be solved by separating the grammars that we are working on. 
I would expect some problems with the positive tests, but it is ok for now because we are not going to merge those changes into the development branch. 
So, let's not add the keywords to a special rule now. All the issues can be fixed later.

Reply:
ok.